### PR TITLE
bugfix text overflow at section item

### PR DESCRIPTION
**Pull request by idanizi-panw**
## Related Issue:
Fixes: https://github.com/demisto/etc/issues/43083

## Screenshots:
- Before:
image

- After:
image

## TODO:
- [x] E2E Test

### CODE REVIEW

**Comment by dorsha** [comment]
Where did this regression came from?

**Comment by idanizi-panw** [comment]
@Bars92 @yonatan-roth I'll be happy for your review... 🙏

File: src/components/Sections/ItemsSection.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { sortByFieldsWithPriority } from '../../utils/sort';
 
 const DESCRIPTION_KEY = 'description';
 const SECTION_ITEM_PADDING = 5;
+const RERENDER_TIMEOUT_MS = 550;
```
Comment (idanizi-panw) [inline src/components/Sections/ItemsSection.js:14]: @yonatan-roth I got to 550ms as the minimum timeout required to render it w/o textoverflowing, but as you might figure - this is a total guess

**Comment by yonatan-roth** [comment]
In general,
`ItemSection` component uses `SectionMarkdown` component under the hood.
We have noticed that there is some issue with height calculations, as the height changes dynamically in some cases by the internal markdown lib.
Looks like this issue already was handled by adding a timeout for the height calculation, this issue introduce a case were we need to increase the timeout (correlated to the text length)

For the long term we have to supply a more robust solution.

**Review by yonatan-roth — APPROVED** [review]
Please notice my comment above.

**Comment by idanizi-panw** [comment]
> In general, `ItemSection` component uses `SectionMarkdown` component under the hood. We have noticed that there is some issue with height calculations, as the height changes dynamically in some cases by the internal markdown lib. Looks like this issue already was handled by adding a timeout for the height calculation, this issue introduce a case were we need to increase the timeout (correlated to the text length)
> 
> For the long term we have to supply a more robust solution.

@yonatan-roth the fact that it's time-dependant itself - code smells 👃 💨 ...

**Comment by idanizi-panw** [comment]
@yonatan-roth  E2E tests complete, I cant make it a test using mocha b.c. the height is determain by the browser. i think selenium is needed for that...

If that's fine for now I will merge this. lmk if ok

**Comment by idanizi-panw** [comment]
@yonatan-roth - we got 👍  by @asafshen

**Pull request merged**